src/main/java/kg/jarkyn/server/Request/ParsedRequest.java

```diff
@@ -0,0 +1,19 @@
+package kg.jarkyn.server.Request;
+
+public class ParsedRequest {
```

Comment:
Is there an `UnparsedRequest` or a `Request` class? Why is it important to note that this is a _parsed_ request?